views/layouts/main.handlebars

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
```

Comment:
its always something super small but I know a lot of time probably went into this. thank you